python/ittapi/compat/pt_region.py

```diff
@@ -0,0 +1,13 @@
+"""
```

Comment:
This file also is not needed anymore since we can replace it with : `from ittapi.pt_region import pt_region as pt_region_create`

Reply:
As discussed with you, the function signatures of pt_region and pt_region_create is different. And pt_region is implemented with the intention to use it as decorator and context manager whereas the pt_region_create is specifically related to NERSC implementation.